source/_components/template.markdown

```diff
@@ -253,38 +264,7 @@ sensor:
         value_template: "{{ states('sensor.power_consumption') }}"
         unit_of_measurement: 'kW'
 ```
-{% endraw %}
-
-### Add Custom Attributes
```

Comment:
I guess this was removed by accident as well...

Reply:
Will fix

Reply:
Completed..